public/docs/swagger.yaml

```diff
@@ -0,0 +1,303 @@
+# this is an example of the Uber API
+# as a demonstration of an API spec in YAML
+swagger: '2.0'
+info:
+  title: HarvestAPI
+  description: Open Data + Agriculture 
+  version: "0.0.0"
+  license:
```

Comment:
@matjames007 I discussed this with @paradoxxjm and I've added a task to JIRA related to determining the appropriate license to be used. The JIRA task includes a note to update this configuration file once that has been decided to I think this can be merged to 'develop' as is if there are no other impediments.